### PR TITLE
Added UG document for AIAssistView Data Binding

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG document for AIAssistView Data Binding

### CODE REVIEW

File: MAUI/AIAssistView/data-binding.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+To add an `AssistItem` to the `ViewModel.AssistItems` collection with specific values for profile details, text, and the `IsRequest` property, follow the steps below:
+
+1. Start by creating an instance of the `AssistItem` class. This item will represent either a user request or a response received from the AI service.
+
+2. The Assistitem has the following members, which provides information for the request/response items,
+
+* `Profile`: Provides information for the user details.
+* `Text`: Describes the text content of the assist item (e.g., the request text from the user or the response text from the AI).
+* `IsRequested`: When the assist item represents a user request, set the `IsRequest` property to `True`. If it's a response item from the AI service, set `IsRequest` to `False`.
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/data-binding.md:22]: API name -> IsRequested or IsRequest.

File: MAUI/AIAssistView/data-binding.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+* `DateTime`: To display item created or received time.
+* `RequestItem`: Describes the data associated with the request item.
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/data-binding.md:24]: Used to hold data (request item) associated with response item. Default value is `null`.

File: MAUI/AIAssistView/data-binding.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+<ContentPage.Content>  
+        <syncfusion:SfAIAssistView x:Name="assistView"
+                       AssistItems  ="{Binding AssistItems}"
+                       RequestCommand="{Binding AssistViewRequestCommand}"/>
+ </ContentPage.Content>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Binding user requests in .NET MAUI AI Assist View (SfAIAssistView)
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/data-binding.md:102]: this topic and content are not needed.
show command declaration and definition in above ViewModel code itself.

**Review by PiruthivirajM — COMMENTED** [review]
Resolve the comments.

File: MAUI/AIAssistView/data-binding.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+# Data Binding in .NET MAUI AI Assist View (SfAIAssistView)
+
+## Generate the assist request & response items
+
+To add an `AssistItem` to the `ViewModel.AssistItems` collection with specific values for profile details, text, and the `IsRequest` property, follow the steps below:
+
+1. Start by creating an instance of the `AssistItem` class. This item will represent either a user request or a response received from the AI service.
+
+2. The Assistitem has the following members, which provides information for the request/response items,
```
Comment (PiruthivirajM) [inline MAUI/AIAssistView/data-binding.md:18]: The assist item

File: MAUI/AIAssistView/data-binding.md (hunk 1 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/data-binding.md:22]: API name is `IsRequested`. Updated names in all places.

File: MAUI/AIAssistView/data-binding.md (hunk 2 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/data-binding.md:24]: Updated

File: MAUI/AIAssistView/data-binding.md (hunk 4 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/data-binding.md:18]: Updated

File: MAUI/AIAssistView/data-binding.md (hunk 3 above)
Comment (Lakshminatarajan) [reply MAUI/AIAssistView/data-binding.md]: Updated

**Review by PiruthivirajM — APPROVED** [review]
This is fine.

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Pull request merged**